stdlib/2.7/getopt.pyi

```diff
@@ -0,0 +1,27 @@
+from typing import List, Tuple
+
+class GetoptError(Exception):
+    opt = ... # type: str
+    msg = ... # type: str
+    def __init__(self, msg: str, opt: str=...) -> None: ...
+    def __str__(self) -> str: ...
+
+error = GetoptError
+
+def getopt(args: List[str], shortopts: str,
+           longopts: List[str]=...) -> Tuple[List[Tuple[str, str]],
+                                             List[str]]: ...
+
+def gnu_getopt(args: List[str], shortopts: str,
+           longopts: List[str]=...) -> Tuple[List[Tuple[str,str]],
+                                             List[str]]: ...
+
+def do_longs(opts: List[Tuple[str,str]], opt: str,
```

Comment:
Do we want this?  It's undocumented. Ditto for long_has_args() and do_short() -- these are all part of the implementation.